apps/cowswap-frontend/src/modules/bridge/updaters/PendingBridgeOrdersUpdater.tsx

```diff
@@ -15,19 +17,24 @@ function PendingOrderUpdater({ chainId, orderUid }: PendingOrderUpdaterProps): R
   const { data: crossChainOrder } = useCrossChainOrder(chainId, orderUid)
   const updateBridgeOrderQuote = useUpdateBridgeOrderQuote()
   const addOrderToSurplusQueue = useAddOrderToSurplusQueue()
+  const fulfillBridgeOrder = useFulfillBridgeOrder()
+  const refundBridgeOrder = useRefundBridgeOrder()
 
   useEffect(() => {
     if (!crossChainOrder) return
 
     const orderUid = crossChainOrder.order.uid
     const isOrderExecuted = crossChainOrder.statusResult.status === BridgeStatus.EXECUTED
+    const isOrderRefund = crossChainOrder.statusResult.status === BridgeStatus.REFUND
 
     if (isOrderExecuted) {
       updateBridgeOrderQuote(orderUid, crossChainOrder.statusResult)
       addOrderToSurplusQueue(orderUid)
-      // TODO: play MOOO sound
+      fulfillBridgeOrder({ chainId, order: crossChainOrder })
```

Comment:
I would avoid using redux. We are in a way to get rid of it.
We can do just `cowSound.play()` here.